scriptmodules/admin/setup.sh

```diff
@@ -595,7 +595,7 @@ function update_packages_gui_setup() {
                 rp_callModule raspbiantools apt_upgrade
             else
                 aptUpdate
-                apt-get -y dist-upgrade
+                apt-get -y full-upgrade
```

Comment:
I think `full-upgrade` is for `apt` not `apt-get` - it works, but isn't in the man file for apt-get (apart from a mention in another parameter), but I think it is equivalent or should be. Doesn't the pinning work with dist-upgrade?

Reply:
https://salsa.debian.org/apt-team/apt/-/blob/1.6.12/cmdline/apt-get.cc#L415

yeah they are the same.

Reply:
(See https://askubuntu.com/questions/770135/apt-full-upgrade-versus-apt-get-dist-upgrade)

Reply:
Yes, it was the `apt` thing that made me change it. But since they're the same, there's no point in changing it - so reverted the changes.